### DISCUSSION TITLE
Discussion 기능을 배워갑니다.

### DISCUSSION THREAD

**Discussion by Kogoon** [learningssik/pull-request#19] [General]
Discussion 기능을 배워갑니다. 

덕분에 이 기능을 처음 알았습니다. 

감사합니다

**Comment by learningssik** [comment]
답변 감사합니다!! 

제 포스팅이 도움이 되었다니 보람찹니다!!  :D

**Comment by learningssik** [comment]
이 글이 도움이 되셨다면 본 레파지토리 우측 상단의 Star를 눌러주세요 ^-^
